src/navigation/reroute.js

```diff
@@ -202,7 +212,7 @@ export function reroute(pendingPromises = [], eventArguments) {
     callCapturedEventListeners(eventArguments);
   }
 
-  function getCustomEventDetail() {
+  function getCustomEventDetail(isBeforeChanges = false) {
```

Comment:
The property names on the events are `newAppStatuses` and `appsByNewStatus`. I think it makes sense for "new" to refer to the status AFTER the routing event. So I'm now passing in the `isBeforeChanges` flag to customize this behavior depending on whether `getAppStatus()` or `appsToMount` should be used to determine which apps to put in which category